### PR TITLE
Fix various spell errors

### PR DESCRIPTION
**Pull request by tossmilestone**
Fix various spell errors

### CODE REVIEW

File: integration-cli/docker_api_containers_test.go [hunk 1]
```diff
@@ -1057,13 +1057,13 @@ func (s *DockerSuite) TestContainerAPIWait(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	defer cli.Close()
 
-	waitresC, errC := cli.ContainerWait(context.Background(), name, "")
+	waitersC, errC := cli.ContainerWait(context.Background(), name, "")
```
Comment (AkihiroSuda) [inline integration-cli/docker_api_containers_test.go]: wait result channel. not typo

Comment (tossmilestone) [reply integration-cli/docker_api_containers_test.go]: @AkihiroSuda updated, PTAL

**Comment by AkihiroSuda** [comment]
Please squash the commits to a single commit.

Also, it would be appreciated if you could collaborate your colleagues to squash all the typo-fix PRs as well.

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**